client/payment-details/summary/index.tsx

```diff
@@ -469,6 +483,63 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 						</div>
 					</div>
 				</div>
+				<div className="payment-details__refund-controls">
+					{ ! charge?.refunded && (
+						<Loadable
+							isLoading={ isLoading }
+							placeholder={ moreVertical }
+						>
+							<DropdownMenu
```

Comment:
I believe we need to control the visibility of the dropdown menu. In my local it stays open after clicking in the 'Refund in full' option. Ideally we should close it. I'd say we can leverage the `isOpen` prop for this purpose.